pkg/cgu/cgu.go

```diff
@@ -273,7 +273,12 @@ func (builder *CguBuilder) Delete() (*CguBuilder, error) {
 		builder.Definition.Name, builder.Definition.Namespace)
 
 	if !builder.Exists() {
-		return builder, fmt.Errorf("cgu cannot be deleted because it does not exist")
+		glog.V(100).Infof("cgu %s in namespace %s does not exist",
+			builder.Definition.Name, builder.Definition.Namespace)
+
+		builder = nil
```

Comment:
I think this should be `builder.Object = nil` instead, the DeleteAndWait test can be changed to have a nil error instead of `fmt.Errorf("error: received nil cgu builder")`

Reply:
Done!